x/gravity/keeper/keeper.go

```diff
@@ -29,45 +30,38 @@ var (
 
 // Keeper maintains the link to storage and exposes getter/setter methods for the various parts of the state machine
 type Keeper struct {
-	paramSpace paramtypes.Subspace
-
 	cdc               codec.BinaryCodec // The wire codec for binary encoding/decoding.
 	bankKeeper        types.BankKeeper
 	StakingKeeper     types.StakingKeeper
 	SlashingKeeper    types.SlashingKeeper
-	DistKeeper        types.DistributionKeeper
+	DistKeeper        distrkeeper.Keeper
 	accountKeeper     types.AccountKeeper
 	ibcTransferKeeper ibctransferkeeper.Keeper
 	evmKeeper         types.EVMKeeper
-
-	storeGetter keeperutil.StoreGetter
+	AddressCodec      address.Codec
+	storeGetter       keeperutil.StoreGetter
 
 	AttestationHandler interface {
-		Handle(sdk.Context, types.Attestation, types.EthereumClaim) error
+		Handle(context.Context, types.Attestation, types.EthereumClaim) error
 	}
+	authority string
```

Comment:
I don't think Paloma uses any authority gated features. Is this implementation enforced by Cosmos?

Reply:
So, params updates on Paloma are currently all done by governance proposals, and our plans are to keep it this way, allowing anyone in the community to propose params udpates.

This `authority` concept is new to Cosmos I believe, and from what I can tell, an update to the params is now a simple message, but must originate from the authority address?

Can we ignore this approach and stick with proposals only?

Reply:
Can you go a bit into detail on this? I'm not aware of this concept on Paloma as of right now. If it was introduced with Cosmos SDK `0.50`, I'm not sure we want to adopt it.

Reply:
https://docs.cosmos.network/v0.50/build/architecture/adr-046-module-params this ADR might provide some info about it

Reply:
It doesn't mention this authority concept. Also, it doesn't appear to have gone past the proposal stage.

Reply:
Hmm, maybe the ADR is outdated. The official docs mention it. I suppose let's take the benefit.